vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -105,3 +105,238 @@ To add a `Figure` to your page:
 
 If the existing figure functions from [`vizro.figures`](../API-reference/figure-callables.md) do not serve your purpose,
 there is always the option to create a [custom Figure](custom-figures.md).
+
+
+### KPI Cards
+A KPI card is a dynamic card that can display a single value. Optionally, a title, icon and reference value can be added.
+It is a common visual component to display key metrics in a dashboard. Vizro supports two pre-defined KPI card functions:
+
+- [`kpi_card`](../API-reference/figure-callables.md#kpi_card): A KPI card with a single value.
+- [`kpi_card_with_reference`](../API-reference/figure-callables.md#kpi_card_with_reference): A KPI card with a single value and a reference value.
+
+#### Existing KPI Card functions
+!!! example "KPI Card Variations"
```

Comment:
```suggestion
- [`kpi_card_with_reference`](../API-reference/figure-callables.md#kpi_card_with_reference): A KPI card with a single value and a reference value.

As described in the [API reference](../API-reference/figure-callables.md) and illustrated in the below example, these functions have several arguments to customise your KPI cards. If you require a level of customisation that is not possible with the built-in functions then you can create a [custom Figure](custom-figures.md).

!!! example "KPI Card Variations"
```
And then delete the whole "Custom KPI Card function" section from page and just do the 3 custom examples we spoke about on the custom figure page.